### PR TITLE
Update Orb Best Practices Guidelines.md

### PR DESCRIPTION
**Pull request by sakshamtaneja21**

### CODE REVIEW

**Review by KyleTryon — APPROVED** [review]
There are some legitimate corrections in here that needed to be addressed. We will accept this PR thank you!

File: Orb Best Practices Guidelines.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ CircleCI orbs are shareable packages of configuration elements, including jobs,
         *   Utilize [semver versioning](https://semver.org/) (x.y.z)
             *   Major: Incompatible changes
             *   Minor: Add new features (backwards compatible)
-            *   Patch: Minor bug fixes, metadata updates, or other safe actions.
+            *   Patch: Minor bug fixes, metadata updates, or other safety actions.
```
Comment (KyleTryon) [inline Orb Best Practices Guidelines.md]: Safe was correct here

**Review by KyleTryon — CHANGES_REQUESTED** [review]

File: Orb Best Practices Guidelines.md (hunk 1 above)
Comment (sakshamtaneja21) [reply Orb Best Practices Guidelines.md]: sure 
will change

Comment (sakshamtaneja21) [reply Orb Best Practices Guidelines.md]: done

**Comment by KyleTryon** [comment]
We appreciate your additions and apologize but this document has been migrated. This can be edited here if you wish to still submit changes. Sorry for the inconvenience. 
https://github.com/circleci/circleci-docs/blob/master/jekyll/_cci2/orbs-best-practices.md

**Pull request closed without merging**